spread.yaml

```diff
@@ -86,7 +86,7 @@ prepare: |
     test -d /home/test || adduser --quiet --disabled-password --gecos '' test
     echo 'test ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
     chown test.test -R ..
-    sudo -i -u test /bin/sh -c "cd $PWD && DEB_BUILD_OPTIONS='nocheck testkeys' dpkg-buildpackage -tc -b -Zgzip"
+    su -l -c "cd $PWD && DEB_BUILD_OPTIONS='nocheck testkeys' dpkg-buildpackage -tc -b -Zgzip" test
```

Comment:
because running things under sudo from root to a different user (as opposed to the other way around) confuses the user detection thing